includes/utility.php

```diff
@@ -76,19 +76,22 @@ function stop_bulk_operation() {
  * @props VIP
  */
 function clear_caches() {
-	global $wpdb, $wp_object_cache;
+	global $wpdb;
 
 	$wpdb->queries = array();
 
-	if ( is_object( $wp_object_cache ) ) {
-		$wp_object_cache->group_ops      = array();
-		$wp_object_cache->stats          = array();
-		$wp_object_cache->memcache_debug = array();
-		$wp_object_cache->cache          = array();
+	// Clear runtime cache to prevent out of memory errors during bulk operations
+	// Use wp_cache_supports() to check for runtime flush capability (WordPress 6.0+)
+	if ( function_exists( 'wp_cache_supports' ) && wp_cache_supports( 'flush_runtime' ) ) {
+		wp_cache_flush_runtime();
+	} else {
+		// Fallback to wp_cache_flush() for older WordPress versions or implementations without runtime support
+		wp_cache_flush();
+	}
 
-		if ( method_exists( $wp_object_cache, '__remoteset' ) ) {
-			$wp_object_cache->__remoteset(); // important
-		}
+	// Reset any custom cache stats if the object cache supports it
+	if ( function_exists( 'wp_cache_stats' ) ) {
+		wp_cache_stats();
 	}
```

Comment:
```suggestion
```

The implementations of `wp_cache_stats()` that I could find use this as a getter, so I don't think this is doing what is intended here.

As it's a relatively small amount of data (number of reads, writes), I think it's fine to leave the data uncleared.